scripts/get-gee-build-environment.sh

```diff
@@ -419,9 +419,9 @@ done
 check_root
 
 if [ -n "$ASSUME_YES" ]; then
-    ASSUME_YES_PACKAGE_MANAGER_PARAMETER="-y"
```

Comment:
I don't think this fixes the issue.  This makes it so that, if you specify `--assume-yes` you don't get assume-yes, and if you disable assume-yes, you get `--assume-yes`.